CDCgov/cdc-open-viz
# Feature/dev 8841

testing steps :
Open the Line/Bar/Area chart.
In the editor panel, under the Left Value Axis, select "Axis Type" and choose "Categorical".
Once "Categorical" is selected, click the new button "Add Category Axis".
You should now see a default label and color bar added to the axis.
Add a category height in numbers, for example, "50".
Update the color to "red".
Add another category and repeat the process.
The last category height can be blank and it will take remaining space axis
Note: For horizontal charts, this feature will not work. Therefore, the "Categorical" axis type is removed.
